Up2dateService/Up2dateClient/Client.cs

```diff
@@ -127,20 +136,27 @@ private bool OnDeploymentAction(IntPtr artifact, DeploymentInfo info)
                 return true;
             }
 
-            WriteLogEntry($"installing...", info);
-            var success = setupManager.InstallPackage(info.artifactFileName);
-            if (!success)
+            var success = true;
+            if (IsSupported(info))
             {
-                WriteLogEntry($"installation failed.", info);
+                WriteLogEntry("installing...", info);
+                success = setupManager.InstallPackage(info.artifactFileName);
+                WriteLogEntry(!success ? "installation failed." : "installation finished.", info);
             }
             else
             {
-                WriteLogEntry($"installation finished.", info);
+                WriteLogEntry("Package is not supported for installing so skipping...", info);
```

Comment:
"not supported" should lead to overall installation fail and negative finished hawkbit result

Reply:
Fixed